### PR TITLE
Compatibility

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,4 +185,13 @@ Console.WriteLine("AB.3(16) = " + y + "(1)");
...
+See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
+
+
+### <a name="contrib"></a>Contribution
+
+We appreciate and welcome any contributors to AngouriMath.
+
+Use pull requests to contribute to it. If you want to regenerate the parser, follow these steps:
+1. Assure you have java on your machine
+2. Run `start ./AngouriMath/antrl_rerun.bat` or `./AngouriMath/antrl_rerun.bat` to regenerate the parser via ANTRL
```
Comment (Happypig375) [inline README.md:197]: Misspelt ANTLR

File: README.md [hunk 2]
```diff
@@ -184,4 +185,13 @@ Console.WriteLine("AB.3(16) = " + y + "(1)");
...
\ No newline at end of file
+See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
+
+
+### <a name="contrib"></a>Contribution
+
+We appreciate and welcome any contributors to AngouriMath.
+
+Use pull requests to contribute to it. If you want to regenerate the parser, follow these steps:
+1. Assure you have java on your machine
```
Comment (Happypig375) [inline README.md:196]: Java runtime to be exact, no meed for the entire JDK

File: README.md [hunk 3]
```diff
@@ -184,4 +185,13 @@ Console.WriteLine("AB.3(16) = " + y + "(1)");
...
-See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
\ No newline at end of file
+See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
+
+
+### <a name="contrib"></a>Contribution
+
+We appreciate and welcome any contributors to AngouriMath.
+
+Use pull requests to contribute to it. If you want to regenerate the parser, follow these steps:
```
Comment (Happypig375) [inline README.md:195]: To be more clear, specify what files constitute the parser.

Comment (WhiteBlackGoose) [reply README.md:195]: ok

**Pull request merged**